code/controllers/subsystem/tts.dm

```diff
@@ -0,0 +1,219 @@
+/*!
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+/proc/tts_alphanumeric_filter(text)
+	// Only allow alphanumeric characters and whitespace
+	var/static/regex/bad_chars_regex = regex(@"[^a-zA-Z0-9 ,?.!'&-]", "g")
```

Comment:
Can you put this in a separate defines file or something? Normally I wouldn't ask for this but GitHub is destroying the entire highlighting of this file and it makes it unreadable for me